### ISSUE TITLE
Input Text Accessibility

### ISSUE THREAD

**Issue by HappyHammer72** [dotnet/Scaffolding#2653]
When I scaffold a Input Text, the accessibility tools show a warning "An element doesn't have an autocomplete attribute"
Would it be sensible to add a default autocomplete="false" to scaffolded Input Texts

**Comment by deepchoudhery** [comment]
will test that out soon and add a default autocomplete="false". thanks for the feedback.
